iphone/Classes/TiUIiOSProxy.m

```diff
@@ -840,5 +840,36 @@ -(id)createFeedbackGenerator:(id)args
 #endif
 #endif
 
+#if defined(USE_TI_UITEXTWIDGET) || defined(USE_TI_UITEXTAREA) || defined(USE_TI_UITEXTFIELD)
+
+MAKE_SYSTEM_STR(CONTENT_TYPE_NAME, UITextContentTypeName);
+MAKE_SYSTEM_STR(CONTENT_TYPE_NAME_PREFIX, UITextContentTypeNamePrefix);
+MAKE_SYSTEM_STR(CONTENT_TYPE_GIVEN_NAME, UITextContentTypeGivenName);
+MAKE_SYSTEM_STR(CONTENT_TYPE_MIDDLE_NAME, UITextContentTypeMiddleName);
+MAKE_SYSTEM_STR(CONTENT_TYPE_FAMILY_NAME, UITextContentTypeFamilyName);
+MAKE_SYSTEM_STR(CONTENT_TYPE_NAME_SUFFIX, UITextContentTypeNameSuffix);
+MAKE_SYSTEM_STR(CONTENT_TYPE_NICK_NAME, UITextContentTypeNickname);
+MAKE_SYSTEM_STR(CONTENT_TYPE_JOB_TITLE, UITextContentTypeJobTitle);
+MAKE_SYSTEM_STR(CONTENT_TYPE_ORGANIZATION_NAME, UITextContentTypeOrganizationName);
+MAKE_SYSTEM_STR(CONTENT_TYPE_LOCATION, UITextContentTypeLocation);
+MAKE_SYSTEM_STR(CONTENT_TYPE_FULL_STREET_ADDRESS, UITextContentTypeFullStreetAddress);
+MAKE_SYSTEM_STR(CONTENT_TYPE_ADDRESS_LINE1, UITextContentTypeStreetAddressLine1);
+MAKE_SYSTEM_STR(CONTENT_TYPE_ADDRESS_LINE2, UITextContentTypeStreetAddressLine2);
+MAKE_SYSTEM_STR(CONTENT_TYPE_ADDRESS_CITY, UITextContentTypeAddressCity);
+MAKE_SYSTEM_STR(CONTENT_TYPE_ADDRESS_STATE, UITextContentTypeAddressState);
+MAKE_SYSTEM_STR(CONTENT_TYPE_ADDRESS_CITY_STATE, UITextContentTypeAddressCityAndState);
+MAKE_SYSTEM_STR(CONTENT_TYPE_SUBLOCALITY, UITextContentTypeSublocality);
+MAKE_SYSTEM_STR(CONTENT_TYPE_COUNTRY_NAME, UITextContentTypeCountryName);
+MAKE_SYSTEM_STR(CONTENT_TYPE_POSTAL_CODE, UITextContentTypePostalCode);
+MAKE_SYSTEM_STR(CONTENT_TYPE_TELEPHONE_NUMBER, UITextContentTypeTelephoneNumber);
+MAKE_SYSTEM_STR(CONTENT_TYPE_EMAIL_ADDRESSS, UITextContentTypeEmailAddress);
+MAKE_SYSTEM_STR(CONTENT_TYPE_URL, UITextContentTypeURL);
+MAKE_SYSTEM_STR(CONTENT_TYPE_CREDIT_CARD_NUMBER, UITextContentTypeCreditCardNumber);
+
+#if IS_XCODE_9
+MAKE_SYSTEM_STR(CONTENT_TYPE_USER_NAME, UITextContentTypeUsername);
```

Comment:
Should be `CONTENT_TYPE_USERNAME`, please also update inside the docs and JIRA example-code.